spec/que/poller_spec.rb

```diff
@@ -275,7 +275,7 @@ def assert_poll(priorities:, locked:)
       assert_equal false, poller.should_poll?
     end
 
-    it "should be false if the number of jobs returned from the last poll was less than the lowest priority request, but the poll_interval has elapsed" do
+    it "should be true if the number of jobs returned from the last poll was less than the lowest priority request, but the poll_interval has elapsed" do
```

Comment:
That was a typo, the test is actually checking `assert_equal true`, which is the expected behavior.